draft-ietf-quic-transport.md

```diff
@@ -1037,8 +1037,17 @@ Servers MUST drop other packets that contain unsupported versions.
 
 Packets with a supported version, or no version field, are matched to a
 connection using the connection ID or - for packets with zero-length connection
-IDs - the address tuple.  If the packet doesn't match an existing connection,
-the server continues below.
+IDs - the address tuple, with the following exception.
+
+A server that uses a non-zero-length connection ID SHOULD handle Initial packets
```

Comment:
Generally this paragraph (also old text) use lower case `connection ID` while the next paragraph does not.

Reply:
:+1: